docs/_news/news13.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: news
+title: New for Codewind 0.13.0
+description: New for Codewind 0.13.0
+keywords: news, new, updates, update, version
+duration: 1 minute
+permalink: news13
+---
+
+## Codewind 0.13.0
+Thursday 11, June 2020
+
+#### ✨ New Features and Highlights for 0.13.0 ✨
+
+**Codewind Quick Guides**
+
+**IDEs**
+##### Eclipse
+- **Important:** Codewind 0.13.0 drops support for Eclipse 2019-03 (4.11). The earliest version of Eclipse that is compatible with Codewind 0.13.0 is Eclipse 2019-06 (4.12).
+- Codewind for Eclipse includes support for [debugging remote projects](https://github.com/eclipse/codewind/issues/1990).
+- Open and view [project log files in the project overview page](https://github.com/eclipse/codewind/issues/3020).
+- In Eclipse 2020-03 (4.15) and later, you can [debug Node.js projects directly in Eclipse](https://github.com/eclipse/codewind/issues/2975). If preferred, use the Codewind preferences to go back to using a Chrome-based browser for debugging.
+- To learn more about why a push registry is required for remote Codewind style projects, [a link to the documentation is included in related dialogs](https://github.com/eclipse/codewind/issues/2992).
+- The Codewind Explorer view displays an indication of project errors and warnings. [Hover over the project to get the details](https://github.com/eclipse/codewind/issues/655).
+
+#### IntelliJ
+- The Codewind plug-in requires IntelliJ IDEA 2020.1.1 as the minimum version.
+
+##### VS Code
+- A [changelog](https://github.com/eclipse/codewind-vscode/wiki/Changelog) file is linked to the VS Code tools `README` file and is updated for Codewind 0.13.0.
```

Comment:
I think we should also include the contents of the changelog here in the news page.

Reply:
Good idea! I'll work on adding the 0.13.0 changelog contents.